plot.py

```diff
@@ -625,6 +629,7 @@ def _check_default_field(self):
         
         This should only occur upon start up with a new file.
         '''
+        #XXX this need re-thinking
```

Comment:
Should display be handling this? may be this should be handle in execute?. Or just don't update if field not in radar and wait for the user to change for a valid field